src/styles/footer.scss

```diff
@@ -0,0 +1,183 @@
+.footer {
+  margin-bottom: 64px;
+
+  @include onTablet {
+    margin-bottom: 120px;
+  }
+
+  &__wrapper {
+    display: grid;
+    grid-template-columns: repeat(2, 130px);
+    column-gap: 20px;
+    justify-content: center;
+
+    @include onTablet {
+      grid-template-columns: repeat(6, 80px);
+      column-gap: 24px;
+    }
+
+    @include onDesktop {
+      grid-template-columns: repeat(12, 1fr);
+      grid-column-gap: 24px;
+    }
+  }
+
+  &__header {
+    font-size: 32px;
+    margin: 100px 0 48px;
+    grid-column: span 2;
+
+    @include onTablet {
+      font-size: 40px;
+      grid-column: span 3;
+    }
+
+    @include onDesktop {
+      grid-area: 1 / 1 / 2 / 5;
+      font-size: 48px;
+      margin: 120px 0 56px;
+    }
+  }
+
+  &__form {
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 6;
+    }
+
+    @include onDesktop {
+      grid-area: 2 / 1 / 5 / 6;
+    }
+  }
+
+  &__field {
+    grid-column: span 2;
+  }
+
+  &__input {
+    background-color: $form-bg-color;
+    width: 100%;
+    height: 48px;
+    border: 2px solid $form-bg-color;
+    border-radius: 8px;
+    font-family: $secondary-font-family;
+    font-size: 14px;
+
+    margin-bottom: 16px;
+    padding-left: 16px;
+
+    transition: all 0.3s;
+
+    &--textarea {
+      height: 100px;
+    }
+
+    @include onDesktop {
+      &:hover {
+        border: 2px solid $hover-border-color;
+      }
+    }
+
+    &:target {
+      border: 2px solid $main-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    background-color: #292929;
+    height: 56px;
+    align-items: center;
+    border-radius: 8px;
+    color: $btn-text-color;
+    margin-top: 16px;
+    margin-bottom: 96px;
+    width: 100%;
+
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 6;
+    }
+
+    @include onDesktop {
+      grid-area: 5 / 1 / 6 / 6;
+      margin-bottom: 0;
+      transition: all 0.3s;
+
+      &:hover {
+        background-color: $btn-hover-bg-color;
+        color: $btn-hover-text-color;
+        border: 2px solid $btn-border-color;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new `border` to the element on hover. Add default transparent border of the same width, and change its color on `:hover`